### PR TITLE
Add clothing toggle to character editor

### PR DESCRIPTION
**Pull request by metalgearsloth**
Thanks to Mellow for the idea.

image

:cl:
- add: Added toggle to hide clothing in the character editor.

### CODE REVIEW

File: Content.Client/Preferences/UI/HumanoidProfileEditor.xaml.cs [hunk 1]
```diff
@@ -112,7 +113,7 @@ public sealed partial class HumanoidProfileEditor : Control
             IEntityManager entityManager, IConfigurationManager configurationManager)
         {
             RobustXamlLoader.Load(this);
-            _random = IoCManager.Resolve<IRobustRandom>();
+            IoCManager.Resolve<IRobustRandom>();
```
Comment (mirrorcult) [inline Content.Client/Preferences/UI/HumanoidProfileEditor.xaml.cs:116]: huh

Comment (metalgearsloth) [reply Content.Client/Preferences/UI/HumanoidProfileEditor.xaml.cs:116]: buh rider told me it was unused and I used its refactor

**Comment by mirrorcult** [comment]
buttons a wee bit big

**Comment by Emisse** [comment]
maybe put button next to clothing tab it took me a bit to find it all the way up there jack and the beanstalk headass

**Comment by metalgearsloth** [comment]
I'll just reformat the whole thing coz it looks like shit anyway

**Comment by mirrorcult** [comment]
im too short man i cant reach the top of the shelf to hit that button

**Comment by dmnct** [comment]
What if you just had to click on the sprite to show/hide clothing

**Comment by Vordenburg** [comment]
> What if you just had to click on the sprite to show/hide clothing

That would need to be communicated to the player in some way for them to learn of it.

**Comment by metalgearsloth** [comment]
image

When I figure out how the fuck to make this align left

**Comment by metalgearsloth** [comment]
image

**Comment by mirrorcult** [comment]
ok yea that is significantly better lol

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**